server/feeders/restaurantfeeder.js

```diff
@@ -74,8 +74,8 @@ var feedOrionRestaurants = function() {
     .catch(function(err) {
       if (err.statusCode == '404') {
         var address = getAddress(attributes);
-        if (typeof attributes.department !== 'undefined') {
-          fwHeaders['fiware-servicepath'] = '/' + attributes.department;
+        if (typeof attributes.department.value !== 'undefined') {
+          fwHeaders['fiware-servicepath'] = '/' + attributes.department.value;
         }
         setTimeout(function() {
          geocoder.geocode({address: address, country: 'Spain'})
```

Comment:
that's certainly an issue. I would prefer that we migrate to an open geo coder, and remove Google dependencies. Please could you open an issue in order to fix that?